src/services/ApiService.unit.spec.ts

```diff
@@ -475,7 +440,28 @@ describe('ApiService', () => {
         it('call the server once', async () => {
           await ApiService.getStatus({ bridge, fromChain, toChain, txHash })
 
-          expect(mockedFetch).toHaveBeenCalledTimes(1)
+          await ApiService.getStatus({
+            bridge: undefined as unknown as string,
```

Comment:
Is the type asserted here just for the test or it's expected in real-case scenario as well.

if `undefined` is an expected value, then the type for this should also be changed in lifi/types to:

```
bridge?: string;
fromChain?: ChainId;
toChain?: ChainId;
```

to avoid assertions.

Reply:
real case.
lifi/types has been changed before. 
```ts
export interface GetStatusRequest {
    txHash: string;
    bridge?: string;
    fromChain?: number | string;
    toChain?: number | string;
}
```